### DISCUSSION TITLE
Post 2.4.90 -> 2.4.100 upgrade, index field type conflict

### DISCUSSION THREAD

**Discussion by geistchevalier** [Security-Onion-Solutions/securityonion#13715] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

6

### RAM

64GB

### Storage for /

500GB

### Storage for /nsm

2TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

In reference to this discussion: [Post 2.4.100 upgrade from 2.4.90, Dashboard menu : System Windows Security errors](https://github.com/Security-Onion-Solutions/securityonion/discussions/13585)

![image](https://github.com/user-attachments/assets/8e5fbae9-9235-46f9-8b59-d7353cc9074d)

I was having issues with opening the dashboard after the update from 2.4.90 to 2.4.100, getting a 500 response (as above), even after running the patch commands. Long story short I went through the aforementioned discussion above and discovered the following issues,
removing the following filters `groupby source.ip | groupby destination.ip` allows the dashboard to be loaded as usual

![image](https://github.com/user-attachments/assets/b361bdcc-a713-45e1-a179-5fa942a6ab73)

![image](https://github.com/user-attachments/assets/cbf69788-adfa-48df-939d-8e9d0d728845)

______________________

going through the mappings for the component templates, I'm guessing that it's the dynamic mapping that's assigning properties like `destination.ip` to the type `keyword` instead of the type `ip` 

![image](https://github.com/user-attachments/assets/44ba8a2c-e0f4-4105-9930-405ab30edac4)
______________________

so I added the mapping for the properties to the appropriate datatype

![image](https://github.com/user-attachments/assets/885cca0e-f08e-4d6d-b9b8-463db4101635)

______________________

My questions are:
1. Do I need to define it in `logs-iis.access@custom` instead of `logs-iis.access@package`? Will these changes persist through SO updates?
2. After rolling over the index to adopt the new template, do I need to close (and delete?) the older indexes that are in conflict? or can I leave it as is?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/issues/13725

This will be fixed in 2.4.110. The above will show you the changes needed. You would then want to rollover your datastream. 

If you want the conflict to go away, you can reindex the affected datastreams.

**Reply by geistchevalier** [reply]
Thanks for linking that, I'll follow the changes in the pull request referenced in there before 2.4.110 comes
